puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/obal.groovy

```diff
@@ -5,7 +5,6 @@ def obal(args) {
     extra_vars = args.extraVars ?: [:]
 
     writeYaml file: extra_vars_file, data: extra_vars
-    archive extra_vars_file
```

Comment:
Why?

Reply:
So, this function is generic, and when you use it twice in one job this file changes. So it was not clear to me what archiving would do for us other than potentially add confusion given it would only archive the last one in the end.

Reply:
Oooh, so it does not archive multiple versions of it? Stupid. Yeah, then it's kinda useless.